modules/gitops-namespace-scoped-operators.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installing-namespace-scoped-operators_{context}"]
+= Installing namepace-scoped Operators
+
+To install namespace-scoped Operators, create and place the `Subscription` and `OperatorGroup` resources of the required Operator in your Git repository.
+
+.Example: Ansible Automation Platform Resource Operator
+
+[source,yaml]
+----
+# ...
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+  name: ansible-automation-platform
+# ...
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: ansible-automation-platform-operator
+  namespace: ansible-automation-platform
+spec:
+  targetNamespaces:
+    - ansible-automation-platform
+# ...
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ansible-automation-platform
+  namespace: ansible-automation-platform
+spec:
+  channel: patch-me
+  installPlanApproval: Automatic
+  name: ansible-automation-platform-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+# ...
+----
+
+[IMPORTANT]
+====
+When deploying multiple Operators using {gitops-title}, you must create only a single Operator group in the corresponding namespace. If more than one Operator group exists in a single namespace, any CSV created in that namespace transition to a `failure` state with the `TooManyOperatorGroups` reason. After the number of Operator groups in their corresponding namespaces reaches one, all the previous `failure` state CSVs transition to `pending` state. You must manually approve the pending install plan to complete the Operator installation.
```

Comment:
You need to expand "CSV". The only standard acronym like that expands to "comma-separated values", a file format, which does not seem to be the case here.